renderCartLine.js

```diff
@@ -0,0 +1,25 @@
+
+export default const renderCartItem = (cartItem, album) => {
```

Comment:
Careful using `export default` with an arrow function. You'll want to do something like this for this line:

`export default renderCartItem = (cartItem, album) => {`

Otherwise you're going to want to do this:

`export default function renderCartItem(cartItem, album) {`